packages/terra-doc-template/src/DocTemplate.jsx

```diff
@@ -92,11 +102,24 @@ const DocTemplate = ({
     exampleHeader = <h1 className={cx('examples-header')}>{exampleHeaderText}</h1>;
   }
 
-  const badge = (
-    <a href={`https://www.npmjs.org/package/${packageName}`}>
-      <img src={`https://badgen.net/npm/v/${packageName}`} alt="NPM version" />
-    </a>
-  );
+  const badge = packageVersion
+    ? (
+      <div className={cx('badge-container')}>
```

Comment:
Let's turn this into a component within the terra-doc-template package. It should make testing easier this way. I'm thinking a new file, `packages/terra-doc-template/src/NpmBadge.jsx` that we can import here.

Reply:
Changed! [af52c67](https://github.com/cerner/terra-core/pull/2571/commits/af52c67081f807649628e8e8f07b4e9957635768)